test/js/utils/accessibility.js

```diff
@@ -0,0 +1,50 @@
+import AxeBuilder from 'axe-webdriverjs'
+const expect = require('chai').expect
+import { Key } from 'selenium-webdriver'
+
+const disabledAccessibilityRules = [
+  'html-has-lang',
+  'landmark-one-main',
+  'region'
+]
+
+const analyzeAccessibility = (driver) =>
+  new Promise((resolve, reject) => {
+    AxeBuilder(driver).disableRules(disabledAccessibilityRules).analyze((error, results) => {
+      if (error) {
+        return reject(console.log(`Accessibility test error: ${error}`))
+      }
+      return resolve(results)
+    }
+  )})
+
+const formatViolationMessages = (violations) =>
+  violations.map(
+    violation =>
+      `\r\n- Rule ID: ${violation.id}
+      \r  ${violation.help} (${violation.nodes.length} elements affected)
+      \r  Help: ${violation.helpUrl}\r\n`).join('')
```

Comment:
I take it the `helpUrl` here be a link to the WAI-ARIA page for the violation found?

Reply:
It's a link to the documentation of the axe-webdriverjs library that gives you tips on how to fix a specific failure. This is an example of `helpUrl` https://dequeuniversity.com/rules/axe/3.2/html-has-lang?application=webdriverjs

Reply:
nice!